recipes/torchkbnufft/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "torchkbnufft" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/torchkbnufft-{{ version }}.tar.gz
+  sha256: 29d93c1299f888f1d7f14f2de5e85d6461785299046a04483abb6134287b716c
+  patches:
+    - patches/0001-conftest-np-2-0.patch
```

Comment:
Is this patch upstream? Also, add a comment to remind you to removed it once a new version with that patch is out.